### PR TITLE
chore: set `PKG_NAME` environment variable

### PR DESCRIPTION
**Pull request by frezbo**
Set `PKG_NAME` environment variable.
This allows to validate extensions.

### CODE REVIEW

File: internal/pkg/convert/graph.go [hunk 1]
```diff
@@ -124,7 +124,6 @@ func (graph *GraphLLB) buildLocalContext() {
 		llb.ExcludePatterns(
 			[]string{
 				"**/.*",
-				"**/" + constants.PkgYaml,
```
Comment (smira) [inline internal/pkg/convert/graph.go:127]: do we need this, or should we inject the package name as the environment variable to the build?

Comment (frezbo) [reply internal/pkg/convert/graph.go:127]: hmm that works too, let me see if it's easier

Comment (frezbo) [reply internal/pkg/convert/graph.go]: fixed

File: internal/pkg/constants/build.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import "os"
 
 // DefaultBaseImage for non-scratch builds.
 // renovate: datasource=docker versioning=docker depName=alpine
-const DefaultBaseImage = "docker.io/alpine:3.19"
```
Comment (smira) [inline internal/pkg/constants/build.go:11]: are we ready for that? :fried_shrimp:

**Review by smira — APPROVED** [review]

File: internal/pkg/constants/build.go (hunk 2 above)
Comment (frezbo) [reply internal/pkg/constants/build.go:11]: these are pkgs, so should be fine

**Pull request merged**